budget app backend/routes/obligatory_route.js

```diff
@@ -0,0 +1,14 @@
+const { obligatory_POST, obligatory_GET, obligatory_PUT, obligatory_DELETE } = require("../controllers/obligatory_controller");
+
+const router = require("express").Router();
+
+// CRUD category
+router.post("/", obligatory_POST)
```

Comment:
What will you do if you need to add few POST requests? Use more meaningful method names (instead of obligatory_POST, obligatory_GET and so on)